guides/docusaurus-starter-guide/rich-content-editing.mdx

```diff
@@ -0,0 +1,29 @@
+---
+_schema: guide-item
+_uuid: "18a28ce5-f9ee-457e-802c-63644e984577"
+_created_at: "2023-07-07 02:40:30 +0000"
+title: "Rich content editing"
+nav_title: "Rich content editing"
+order: 5
+published: true
+image: "/documentation/static/CloudCannonDocumentationog.jpg"
+description: "This guide will walk through the steps required to get your Docusaurus site built, editable and live on CloudCannon."
+tags: []
+related_articles: null
+related_links: []
+---
+
+When writing content with Docusaurus, you can use MDX to build rich, interactive pages by including your React components
+directly alongside your Markdown content. With CloudCannon Snippets you can integrate your
+MDX files and React components with the CloudCannon Content Editor. This means you get clean and intuitive Markdown
+editing with the power to create and edit structured components — all without
+touching the underlying code.
+
+<comp.DocsImage
+  path="/static/docusaurus-starter-guide/content-editor.png"
+  alt="Screenshot of CloudCannon's content editor showing structured Snippets alongside Markdown content"
+  type="screenshot"
+/>
+
+To get started with Snippets in Docusaurus, you can follow this guide on setting up the builtin
+[MDX Component](/documentation/articles/editing-with-mdx-components/) and [Docusaurus](/documentation/articles/editing-with-docusaurus/) Snippets.
```

Comment:
The "editing-with-docusaurus" link doesn't seem to go anywhere

Reply:
This just needed a pull from main to get the new docusaurus snippets docs